specification/compute/resource-manager/Microsoft.Compute/stable/2019-07-01/compute.json

```diff
@@ -7770,7 +7770,7 @@
         },
         "osProfile": {
           "$ref": "#/definitions/OSProfile",
-          "description": "Specifies the operating system settings for the virtual machine."
+          "description": "Specifies the operating system settings used while creating the virtual machine. Some of the settings cannot be changed once VM is provisioned."
```

Comment:
Can you be more specific by explicitly saying which settings cannot be changed?

Reply:
I have added the statement about "a property value not being updatable" to some properties individually and some already exist. 

Since the same per OSProfile property description is shared for Put and Patch, adding this  "**Some of the settings cannot be changed once VM is provisioned.**" statement.